addons/event/tests/test_event_mail_schedule.py

```diff
@@ -402,3 +402,22 @@ def test_archived_event_mail_schedule(self):
 
         # check that scheduler is not executed
         self.assertFalse(event_prev_scheduler.mail_done, 'event: reminder scheduler should should have run')
+
+    def test_chatter_message_body(self):
```

Comment:
actually this test is a bit strange. I would test directly the "send_mail" method of mail.template, as it is what we are modifying. You can modify or add a small test in test_mail/tests/test_mail_template.py file in 'TestMailTemplate' class. Seems better located in that module.